### PR TITLE
deps: bump SDK to 0.46-rc1

### PR DESCRIPTION
**Pull request by crodriguezvega**

### CODE REVIEW

**Comment by yaruwangway** [comment]
hi @crodriguezvega , is this branch ready to use ? would like to use this branch to integrate into gaia. 😄

**Comment by adizere** [comment]
> Yes, this is ready for now. I haven't tested it though; not sure if hermes has finalized the support for SDK 0.46...

Currently we have a WIP branch that is compat. with Gaia Rho. I think it's this one: https://github.com/informalsystems/ibc-rs/pull/2213. Note that this branch is _not_ compatible with SDK 0.45.

**Comment by yaruwangway** [comment]
thanks @adizere ! 👍

File: go.mod [hunk 1]
```diff
@@ -5,118 +5,143 @@ go 1.17
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.1
-	github.com/cosmos/ibc-go/v3 v3.0.0
+	github.com/cosmos/cosmos-sdk v0.46.0-rc1
+	github.com/cosmos/ibc-go/v3 v3.0.0-alpha1.0.20220530074104-c7c115244bad
```
Comment (damiannolan) [inline go.mod:9]: Does this refer to [this ibc-go branch](https://github.com/cosmos/ibc-go/tree/carlos/upgrade-sdk-0.46-tendermint-0.35)?

Comment (crodriguezvega) [reply go.mod:9]: No, actually no. It is pointing to a branch that is now merged into that one, so I do need to update this PR. Thanks for pointing this out!

**Pull request closed without merging**